internal/profiles/viper.go

```diff
@@ -380,6 +380,15 @@ func GetOptionValue(opt options.Option) (pFlagValue string, err error) {
 	return "", fmt.Errorf("failed to get option value: no value found: %v", opt)
 }
 
+func MaskValue(value string) string {
+	if value == "" {
+		return ""
+	}
+
+	maskLength := min(len(value), 16)
```

Comment:
We shouldn't print the number of characters at all if we intend to make the output totally masked, especially for something like a password.

Reply:
Updated mask function to always return 8 asterisks.